_usage/security.md

```diff
@@ -57,21 +55,18 @@ installation:
 - `tempdb_dbo`
 - `tempdb_guest`
 
-These users are MS SQL-specific and try to mimic what the target database is
-supposed to look like. 
+Moreover, there will be a <code>*dbname*&lowbar;db&lowbar;owner</code>
+and a <code>*dbname*&lowbar;dbo</code> for every database you create in
+Babelfish.
 
-In addition to that, all users that are normally available in PostgreSQL are
-available as well:
+These roles are there to implement the SQL Server ownership structure.
```

Comment:
These roles implement the SQL Server ownership structure.

Reply:
Yes, that is better. Done in 66c1e14bbb6027cf6bda77acf5c6e66779dfda5d.